PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/Invisicam.rbxmx

```diff
@@ -165,6 +164,21 @@ end
 
 -- Update. Called every frame after the camera movement step
 function Invisicam:Update()
+	-- Make sure we still have a Torso
```

Comment:
Do we need to do this every tick() or just when we noticed when the Torso's ancestry has changed?